unit/deserialize_test.go

```diff
@@ -218,7 +219,7 @@ Description= words here `),
 Description= words here \
   `),
 			[]*UnitOption{
-				&UnitOption{"Unit", "Description", "words here"},
+				&UnitOption{"Unit", "Description", "words here \\"},
```

Comment:
not sure about this test. when we serialize this again, wouldn't this lead to the next UnitOption being merged with this one by Systemd?